modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -161,6 +161,14 @@ export default {
         return {};
       }
     },
+    // not used, but we need to keep it here to avoid
```

Comment:
Will there be any practical problems for third parties who have already overwritten our various components or will they jut see some `[object Object]` in the DOM? Obviously they will not get the benefit of the feature, but that is OK since it is a new feature.

Reply:
We already have `[object Object]` in the core in some cases. It doesn't cause any error. Vue is just doing `toString` for every unknown prop and add it as an HTML attribute by default. Custom components can decide if they allow custom attributes passed as props (a good idea almost all the time) or just reject the unknowns. We don't have many options here.

Reply:
However, usually custom field implements the input mixin, which defines the new prop.